test/runner.zig

```diff
@@ -351,7 +352,7 @@ var is_fuzz_test: bool = undefined;
 extern fn fuzzer_set_name(name_ptr: [*]const u8, name_len: usize) void;
 extern fn fuzzer_init(cache_dir: FuzzerSlice) void;
 extern fn fuzzer_init_corpus_elem(input_ptr: [*]const u8, input_len: usize) void;
-extern fn fuzzer_start(testOne: *const fn([*]const u8, usize) callconv(.C) void) void;
+extern fn fuzzer_start(testOne: *const fn([*]const u8, usize) callconv(.c) void) void;
```

Comment:
Please avoid making too many cosmetic changes like this. We will need to recheck them every time we need to update this file to the Zig baseline.

Reply:
That one was not intentional, I actually trying to make it the way it was, but I guess I misinterpreted the diff

Reply:
```suggestion
extern fn fuzzer_start(testOne: *const fn([*]const u8, usize) callconv(.C) void) void;
```